src/main/java/net/imagej/ops/transform/scaleView/DefaultScaleView.java

```diff
@@ -70,6 +72,10 @@
 	@Parameter
 	private InterpolatorFactory<T, RandomAccessible<T>> interpolator;
 
+	@Parameter(required = false)
+	private OutOfBoundsFactory<T, RandomAccessibleInterval<T>> outOfBoundsFactory =
+		new OutOfBoundsMirrorFactory<>(OutOfBoundsMirrorFactory.Boundary.SINGLE);
```

Comment:
What if the user passes in `null`? Then won't that overwrite the default here? Did you try it?